dipy/reconst/dti.py

```diff
@@ -1905,7 +1914,65 @@ def design_matrix(gtab, dtype=None):
     return -B
 
 
-def quantize_evecs(evecs, odf_vertices=None):
+def _quantize_evecs_parallel(data, odf_vertices, v, nbr_processes):
+    if nbr_processes is -1:
+        try:
+            nbr_processes = cpu_count()
+        except NotImplementedError:
+            warn("Cannot determine number of cpus. \
```

Comment:
Thanks for noticing. I fixed this, and extended this based on http://stackoverflow.com/a/13544896/3532933, but I am not sure how to test all this, because it seems that we would end up here only in some rare circumstances.